build.gradle.kts

```diff
@@ -30,6 +30,7 @@ dependencies {
   implementation("com.fasterxml.uuid:java-uuid-generator:5.1.0")
   implementation("org.springdoc:springdoc-openapi-starter-webmvc-ui:2.6.0")
   implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.17.2")
+  implementation("io.netty:netty-common:4.1.115.Final")
```

Comment:
Are you sure we need to so this.  The SQS library should be fixed not our service

Reply:
@Mjwillis 

We get the dependency from `hmpps-kotlin-spring-boot-starter` which itself depends on 

https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/3.3.5

which is the latest available. You are right that we should raise the issue in `hmpps-kotlin-spring-boot-starter` and suggest the patch above so that everyone benefits from it. 

In the short term and until a new version of hmpps-kotlin-spring-boot-starter is available and fixes it, I think we should still apply this small change so that we don't get the alerts reminding us of it all the time.